docs: add intro to abstract documentation (#38)
AbstractSDK/abstract: framework/docs/src/3_framework/3_account_abstraction.md

# Account Abstraction

!todo

> An Abstract Account is a smart-contract wallet developed by Abstract. That means that the smart-contract is capable of holding tokens and interacting with other smart-contracts. The contract's ownership structure is customizable to fit the needs of users or applications. It is designed to be highly programmable, allowing developers to build and distribute complex applications on top of it.

Reviewer: ```suggestion
An **Abstract Account** is an extensible smart-contract wallet. That means that the smart-contract is capable of holding tokens and interacting with other smart-contracts. The contract's ownership structure is customizable to fit the needs of users or applications. It is designed to be highly programmable, allowing developers to build, distribute, and integrate complex applications on top of it.
```
